core/templates/dev/head/collection_player/collection_player.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
+  {{ collection_title }}
+{% endblock subtitle %}
+
+{% block header_js %}
+  {{ super() }}
+  {% if collection_id %}
+    <script type="text/javascript">
+      GLOBALS.collectionId = JSON.parse('{{collection_id|js_string}}');
+      GLOBALS.isLoggedIn = Boolean(JSON.parse('{{is_logged_in|js_string}}'));
+    </script>
+
+    <style>
+      html, body {
+        background-color: #eee;
+      }
+    </style>
+  {% endif %}
+
+  {{dependencies_html}}
+{% endblock header_js %}
+
+{% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      <span class="protractor-test-collection-header" itemprop="description">{{collection_title}}</span>
+    </li>
+  </ul>
+{% endblock navbar_breadcrumb %}
+
+{% block content %}
+  <div ng-controller="CollectionPlayer">
+    <div ng-if="collection" class="oppia-collection-player-tiles-area">
+      <div class="oppia-collection-player-tiles-container">
+        <div ng-if="!hasFinishedCollection()" class="oppia-collection-player-tiles-header">
+          <span ng-if="!hasStartedCollection()">Begin by playing:</span>
+          <span ng-if="hasStartedCollection()">Continue where you left off:</span>
+        </div>
+        <div ng-if="hasFinishedCollection()" class="oppia-collection-player-tiles-header">
+          <span>You have finished the collection! Feel free to replay any explorations below.</span>
+        </div>
+
+        {% if not is_logged_in %}
+          <span class="oppia-collection-player-small-text">You must login to save your progress.</span>
+        {% endif %}
+
+        <div ng-repeat="collection_node in getDisplayedCollectionNodes()">
+          <md-card layout="row" class="oppia-collection-player-tile protractor-test-collection-player-tile">
+            <a ng-href="/explore/<[collection_node.exploration.id]>?collection_id=<[collectionId]>" class="oppia-collection-player-tile-container-link" style="text-decoration: none;">
+              <div ng-class="{'oppia-collection-player-finished-exploration-tile': collection_node.exploration.completed}" class="oppia-collection-player-tile-contents">
+                <span class="oppia-collection-player-tile-image-container">
+                  <img class="oppia-collection-player-tile-image" ng-src="/images/gallery/exploration_background_teal_small.png">
+                </span>
+
+                <div class="oppia-collection-player-tile-details">
+                  <span class="oppia-collection-player-tile-title protractor-test-gallery-tile-title">
+                    <[collection_node.exploration.title]>
```

Comment:
I do think 'last updated' and 'category' are noisy and probably not necessary here. Custom objectives seem like a reasonable idea to me but I'd rather tackle that in a separate pass, possibly after we've created some actual collections to see how necessary this is and whether it would be confusing to learners/creators to have different objectives in different contexts.

Re the pre-requisites issue, I agree with the problem but am not sure about the solution -- I think this needs a more thorough design. @BenHenning, did you get a chance to sync with @xtyang on this? I feel like we would want to visually illustrate the metaphor of 'unlocking'/'new access' somehow -- like, in games where when you finish a level, you're taken back to the menu screen, and a little animation shows a line extending from the thing you just played to the new thing that opens up, and the new thing appears in the list of 'next explorations'. Without some kind of animation like this, I'm not sure how you can convey this well.